.github/workflows/ofac.yaml

```diff
@@ -0,0 +1,53 @@
+name: Injective list - Fetch OFAC wallets
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "30 2 * * *" # Every day at 02:30 PM
+  push:
+    branches:
+      - master
+
+jobs:
+  generate:
+    if: ${{ !contains(github.event.commits[0].message, 'fetch ofac addresses') }}
```

Comment:
This workflow will trigger on regular commits too, should we enable it to run only on cron?